deployment.go

```diff
@@ -99,6 +100,12 @@ func (d *Deployment) restartServices(services []string, failuresPermitted, timeo
 		return nil
 	}
 
+	// Don't restart services when a lock is present
+	if _, err := os.Stat("/etc/service/dont-restart"); err == nil {
```

Comment:
Did this (creating `/etc/sv-rollout`) in cookbooks since it seemed super hard to do it here